hipify-backward-compat.cmake

```diff
@@ -122,11 +47,7 @@ function(create_binary_symlink)
                   COMMAND ${CMAKE_COMMAND} -E create_symlink
                   ../../bin/${file_name} ${HIPIFY_WRAPPER_BIN_DIR}/${file_name})
 endfunction()
-#Creater a template for header file
-create_header_template()
-#Use template header file and generater wrapper header files
-generate_wrapper_header()
```

Comment:
clang header files wrapping won't be introduced in a separate change, right? It looks like there is no need in it for backward compatibility.

Reply:
Correct, there won't be any separate checkin for header wrappers and no need for backward compatibility for the same 
As per File Reorganization , the include files should be in /opt/rocm-ver/include/hipify.  The current patch doesn't have that   change
These changes need to be incorporated  before merging the patch. Will sync up on this